templates/vanilla/package.json

```diff
@@ -13,6 +13,8 @@
   },
   "devDependencies": {
     "@latticexyz/cli": "link:../../packages/cli",
+    "@types/debug": "4.1.7",
+    "@types/prettier": "2.7.2",
```

Comment:
likewise here, I don't think we're using any of these packages directly, but getting this via the cli dep?

Reply:
cli is not relevant to this diff.
prettier and debug are added as dependencies at the workspace root because both contracts and client require them. Similarly, typescript is also a dependency in the workspace root.

Reply:
And both contracts and client use the TS files from common (via typesVersions), and common uses prettier and debug.